compiler/erlang_check.erl

```diff
@@ -245,29 +244,26 @@ check_module(File) ->
             report,
             % By adding debug_info, we ensure that the output of xref:m will
             % contain the caller MFAs too.
-            debug_info,
-            {i, AbsDir ++ "/include"},
-            {i, AbsDir ++ "/../include"},
-            {i, AbsDir ++ "/../../include"},
-            {i, AbsDir ++ "/../../../include"}],
```

Comment:
Let's not remove these lines:

    {i, AbsDir ++ "/include"},
    {i, AbsDir ++ "/../include"},
    {i, AbsDir ++ "/../../include"},
    {i, AbsDir ++ "/../../../include"}],

Commit 443fb436586c09140419c02543e29f650e6f4b02 suggest that they were added for a reason. (I don't have a project layout for which these are necessary, but I'm sure that if we remove them, we will break someone's setup.)